pkg/scaffold/v2/types.go

```diff
@@ -66,6 +66,7 @@ import (
 type {{.Resource.Kind}}Spec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
+	Foo string ` + "`" + `json:"foo,omitempty"` + "`" + `
```

Comment:
I'd suggest add a new line and a short description of Foo.
Change will affect what appear in the CRD e.g. testdata/project-v2/config/crd/bases/crew.testproject.org_admirals.yaml
We may want something like:
```
// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
// Important: Run "make" to regenerate code after modifying this file

// Foo is an example field of Spec.
Foo string ` + "`" + `json:"foo,omitempty"` + "`" + `
```

Reply:
Great call, will push change now.

Reply:
Spec now gen'd is 

```yaml
        properties:
            foo:
              description: Foo is an example field of Admiral. Edit Admiral_types.go
                to remove/update
              type: string
```